optimization.py

```diff
@@ -63,18 +70,27 @@ def create_optimizer(loss, init_lr, num_train_steps, num_warmup_steps, use_tpu):
       beta_2=0.999,
       epsilon=1e-6,
       exclude_from_weight_decay=["LayerNorm", "layer_norm", "bias"])
+  
+  if use_hvd:
```

Comment:
I think they are mutually exclusive, but not either/or relationship. For example, it can be non-TPU and non-Horovod case where there is a single GPU used.